exonum/src/events/mod.rs

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![allow(missing_debug_implementations, missing_docs)]
```

Comment:
Is `missing_docs` really needed here? The module is "doc hidden" anyway. :thinking: 

Plus I'm thinking about allowing "missing debug implementations": why not just add `derive(Debug)` everywhere?

Reply:
Yep, `missing_docs` barks even in `doc(hidden)` module.

About `derive(Debug)` - I tried, but there're some structs from `snow` that don't implement `Debug`.

Reply:
I guess you can create issue or even PR in snow. Maintainer reacts pretty fast.

Reply:
@popzxc will do :) But `allow(missing_debug_implementations)` will be removed in another PR

Reply:
@ozkriff You are going to create an issue for removing `allow(missing_debug_implementations)`, right?

Reply:
@DarkEld3r yep https://jira.bf.local/browse/ECR-1818